src/Microsoft.PowerShell.ConsoleHost/host/msh/Telemetry.cs

```diff
@@ -50,14 +73,18 @@ private static void SendTelemetry(string eventName, Dictionary<string,string>pay
             try
             {
                 // if the semaphore file exists, try to send telemetry
-                if (Utils.NativeFileExists(TelemetrySemaphoreFilePath))
+                var Enabled = Utils.NativeFileExists(TelemetrySemaphoreFilePath) && !GetEnvironmentVariableAsBool(TelemetryOptoutEnvVar, false);
+
+                if ( ! Enabled )
```

Comment:
for consistency (like line 76 above), remove the whitespace between `!` and `enabled`

Reply:
haha, that is the WordPress coding style.. Polyglots: knowledge of many things, master of none. I'll fold that into the same commit as the camelCased `Enabled` variable.